packages/webpack/nx-typescript-webpack-plugin.ts

```diff
@@ -0,0 +1 @@
+export { NxTsconfigPathsWebpackPlugin } from './src/plugins/nx-typescript-webpack-plugin/nx-tsconfig-paths-webpack-plugin';
```

Comment:
Can we tighten up the path a bit to `@nx/webpack/tsconfig-paths-plugin`?

Should look like:
```
import { NxTsConfigPathsPlugin } from '@nx/webpack/tsconfig-paths-plugin';
```